### ISSUE TITLE
Allow overriding dacpac filename for a package reference

### ISSUE THREAD

**Issue by jmezach** [rr-wfm/MSBuild.Sdk.SqlProj#396]
When adding a `PackageReference` to a project, MSBuild.Sdk.SqlProj currently assumes that this package contains a `.dacpac` in the `tools` folder which has the same name as the package. So for example, if I add a `<PackageReference Include="MyPackage">` to the project file, it will look for a file called `tools\MyPackage.dacpac` and use that as a reference. If such a file does not exist, the package reference is essentially ignored.

While that is a fair assumption that works for most scenarios, and is in fact also the result of doing a `dotnet pack` on an MSBuild.Sdk.SqlProj project, there are some scenarios where this doesn't hold up. One example is the newly released [Microsoft.SqlServer.Dacpacs](https://www.nuget.org/packages/Microsoft.SqlServer.Dacpacs) package, which holds the `.dacpac` for the `master` and `msdb` databases. Ideally you should be able to reference those in an `MSBuild.Sdk.SqlProj` project, but you can't do that currently.

I would therefore like to introduce the possibility to override the name of the `.dacpac` on a per `PackageReference` basis. For example, something like this:

```xml
<PackageReference Include="Microsoft.SqlServer.Dacpacs" DacpacPath="tools\master.dacpac" />
```

**Comment by jmezach** [comment]
@ErikEJ @jeffrosenberg Any thoughts on this?

**Comment by jmezach** [comment]
Yeah, I guess we could just stick with `DacpacFilename` instead and assume it is in the `tools` folder. Not sure if somebody would want to put it into a different folder.

**Comment by jeffrosenberg** [comment]
Makes sense to me, we can always add a path parameter if that comes up at some point

**Comment by ErikEJ** [comment]
`<ItemGroup> 
     <PackageReference Include="Microsoft.SqlServer.Dacpacs" Version="160.0.0" /> 
       <DacpacName>msdb</DacpacName> 
     </PackageReference> 
   </ItemGroup> 
 </Project>`

**Comment by ErikEJ** [comment]
This is how MS does it.

**Comment by jmezach** [comment]
Yeah, that looks good to me. I'll see if I can implement something like that for MSBuild.Sdk.SqlProj.

**Comment by ErikEJ** [comment]
Fxing this will fix #64

**Comment by ErikEJ** [comment]
@jmezach Notice that there are multiple packages published: https://github.com/microsoft/DacFx/issues/83#issuecomment-1431754035

**Comment by jmezach** [comment]
This is now released in version 2.5.0.
